### ISSUE TITLE
[Bug] Equipable Twilight Forest Charms

### ISSUE THREAD

**Issue by GoldenTotem** [RaveTr/Twilight-Forest-Tweaks#1]
Minecraft 1.16.5  Forge 36.2.35  TFC 1.0.1

Charm of Keeping doestn't work in Curios slots.
Charm of live may clear your inventory when it works in Curios slots.

**Comment by RaveTr** [comment]
I'll rewrite the mod under the name "Twilight Forest Tweaks" and fix those bugs, as well as add new features and stuff. Thanks for bringing them to my attention.

**Comment by GoldenTotem** [comment]
> I'll rewrite the mod under the name "Twilight Forest Tweaks" and fix those bugs, as well as add new features and stuff. Thanks for bringing them to my attention.

Hi, is this mod available now?

**Comment by RaveTr** [comment]
Totally forgot I had this project. I'll get to work sometime in a few days (or once I'm free from exams lolol).

**Comment by GoldenTotem** [comment]
> Totally forgot I had this project. I'll get to work sometime in a few days (or once I'm free from exams lolol).

Hi, is this mod available now? I am keeping wait for you.

**Comment by RaveTr** [comment]
Yeah, life once again hit. Really sorry to have kept you waiting this long (seriously - I honestly thought it was dead by now lmao).

I have a number of things I need to get done before getting back to work on this. I'll get back to work on it soon, hopefully, but no ETAs.

**Comment by GoldenTotem** [comment]
> Yeah, life once again hit. Really sorry to have kept you waiting this long (seriously - I honestly thought it was dead by now lmao).
> 
> I have a number of things I need to get done before getting back to work on this. I'll get back to work on it soon, hopefully, but no ETAs.

Life is going on, and hope you will have a good day. If you are going to restart this project, do not forgot to mention it.
